examples/compute_noop.py

```diff
@@ -59,7 +59,8 @@
 
 # %% Create device
 # Create device and shader object
-device = wgpu.utils.get_default_device()
+adapter = wgpu.gpu.request_adapter(power_preference="high-performance")
+device = adapter.request_device(required_features=[wgpu.FeatureName.timestamp_query])
```

Comment:
Let's keep the compute_noop example minimal if we can. Can you move your timestamp query example to its own example module?

Reply:
I moved it to `examples/compute_timestamps.py`